.travis.yml

```diff
@@ -45,14 +45,22 @@ matrix:
       language: generic
       env:
         - MB_PYTHON_VERSION=3.5
-        - MACOSX_DEPLOYMENT_TARGET=10.6 # since pandas PR24274, mac builds default to 10.9
     - os: osx
       language: generic
```

Comment:
@robbuckley is there a reason this 3.6 job is still here? also the 3.5 wheel doesn't have MB_PYTHON_OSX_VER=10.9 is that ok?

Reply:
the 3.6 / 10.6 job can go. The 3.5 job is built for 10.6 / dual arch because python.org only provide 10.9 / 64b-only builds of CPython for 3.6.5 and up, and thats what multibuild uses to build the wheels. So if you need to provide a 3.5 wheel, it will have to be 10.6 / dual arch, with the way multibuild is currently implemented